src/lib/sitemap/helpers.ts

```diff
@@ -28,7 +28,11 @@ export const makeSitemapField = ({
 	changefreq,
 }: SitemapFactoryInput): SitemapElement => {
 	return {
-		loc: `${__config.dev_dot.canonical_base_url}${slug}`,
+		// Regex: remove trailing or leading slash from slug
+		loc: `${__config.dev_dot.canonical_base_url}/${slug.replace(
+			/^\/|\/$/g,
+			''
+		)}`,
```

Comment:
I think if we're gonna use a regex here, let's use `URL` as @kendallstrautman mentioned:
```suggestion
		loc: new URL(slug, __config.dev_dot.canonical_base_url).toString(),
```

Reply:
Ah I see, I was thinking it may have been a HashiCorp API for URLs 😆. Added! Thanks @BRKalow & @kendallstrautman